src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/VSSemanticTokensDeltaResponse.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Runtime.Serialization;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor
+{
+    /// <summary>
+    /// Roslyn supports frozen compilations and may only send us back partial tokens
```

Comment:
I wouldn't bind this comment to this only being "frozen" compilations. This also enables us to support multiple colorization passes too in order to improve time-to-colorization.